compiler/rustc_borrowck/src/lib.rs

```diff
@@ -465,16 +473,6 @@ fn do_mir_borrowck<'tcx>(
 
     mbcx.report_move_errors();
 
-    // If requested, dump polonius MIR.
-    polonius::dump_polonius_mir(
```

Comment:
I put the call here because the dump will use the dataflow results just above

Reply:
as in, moving this is wrong?

it does not access `flow_results`  or the `mbcx`, so how does the code above impact this output :thinking:

Reply:
It doesn’t on master right now but will, when I open the PR to add the full Polonius debugger to its MIR dump: it shows the loans going out of NLL scope for example.

No worries, I will move it back if it needs to, but I also don’t know if I’ll open that PR before needing to rework active loans away from these dataflow results, so it’s fine.